content/agent/proxy.md

```diff
@@ -205,7 +205,7 @@ Assuming that the supervisor file is found at `<SUP_FILE>`
 sed -i 's/ddagent.py/ddagent.py --sslcheck=0/' <SUP_FILE>
 ```
 
-For the Windows Agent (Starting from Agent version 3.9.2), edit your configuration file `datadog.conf` and add this option:
+For the Windows Agent, edit your configuration file `datadog.conf` and add this option:
```

Comment:
On Agent6 this is `datadog.yaml`

Reply:
You are right, but here it's in the Agent v5 section.